### PR TITLE
One file script to verify BERT working on ROCm

### PR DESCRIPTION
**Pull request by c0redumb**
Please check this simple one file script works. It is for customer to easily verify that BERT works with ROCm on their AMD hardware.

### CODE REVIEW

File: scripts/run_model_in_rocm_docker.sh [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+# Get the folders
+SCRIPTPATH=$(dirname $(realpath $0))
+CODE_DIR=$SCRIPTPATH/..
+CODE_DIR_INSIDE=/data/code
+
+# Pull the docker image
+echo 
+echo "=== Docker pulling image $IMAGE and start container ==="
+docker pull $IMAGE
+CTNRNAME=RunDocker
```
Comment (micmelesse) [inline scripts/run_model_in_rocm_docker.sh]: Rename container to something more clear. Maybe ROCMDockerContainer.

File: scripts/run_model_in_rocm_docker.sh [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+# Preparing training data
+echo 
+echo "=== Preparing training data ==="
+
+# Generate the training data
+DATA_SOURCE_FILE_PATH=sample_text.txt
+DATA_SOURCE_NAME=$(basename "$DATA_SOURCE_FILE_PATH")
+
+SEQ=128
```
Comment (micmelesse) [inline scripts/run_model_in_rocm_docker.sh]: Any reason why we are using Seq 128?

File: scripts/run_model_in_rocm_docker.sh [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+# Perform training
+echo 
+echo "=== Training BERT ==="
+CUR_TRAINING=${DATA_SOURCE_NAME}_ba${BATCH}_seq${SEQ}
+mkdir -p $TRAIN_DIR/$CUR_TRAINING
+
+TRAIN_STEPS=200
+WARMUP_STEPS=50
+BATCH=8
+LEARNING_RATE=2e-5
```
Comment (micmelesse) [inline scripts/run_model_in_rocm_docker.sh]: Any reason why this learning rate?

Comment (c0redumb) [reply scripts/run_model_in_rocm_docker.sh]: It makes no difference if we are running only 200 steps. This is just to demonstrate that the code runs on ROCm.

File: scripts/run_model_in_rocm_docker.sh (hunk 2 above)
Comment (c0redumb) [reply scripts/run_model_in_rocm_docker.sh]: It seems to be an easy choice for most of the hardware without getting close to the edge.

File: scripts/run_model_in_rocm_docker.sh (hunk 1 above)
Comment (c0redumb) [reply scripts/run_model_in_rocm_docker.sh]: It is just the name for the container, which only runs for a few minutes and it is gone. But feel free to change it to anything that makes a better sense.

**Pull request merged**